tests/integration/server/api/users_api_test.py

```diff
@@ -426,3 +426,43 @@ def test_reject_users_proposals():
 
         assert admin
         assert not non_admin
+
+
+def test_update_user_password():
+    """Test that an admin user can change a user's password."""
+    user = {
+        "name": "nadia five",
+        "username": "nadia5",
+        "password": "test11",
+        "email": "nadia5@test.com",
+    }
+    with requests.Session() as session:
+        created_user = create_test_user(session, user)
```

Comment:
I might be missing something, but how is the nadia5 user an Next Admin? I don't see the user being added to the NextAdmins role after being created here.